src/lightning_lite/strategies/launchers/subprocess_script.py

```diff
@@ -24,7 +24,7 @@
 from lightning_lite.plugins.environments.cluster_environment import ClusterEnvironment
 from lightning_lite.strategies.launchers.base import _Launcher
 
-_HYDRA_AVAILABLE = RequirementCache("hydra")
+_HYDRA_AVAILABLE = RequirementCache("hydra-core")
```

Comment:
This was a bug, we forgot to update it. It is also like this in the PL version. I added a test. 
With this, the PR is complete I think.